nsone/__init__.py

```diff
@@ -116,6 +116,16 @@ def loadZone(self, zone, callback=None, errback=None):
         zone = nsone.zones.Zone(self.config, zone)
         return zone.load(callback=callback, errback=errback)
 
+    def searchZone(self, zone, q=None, has_geo=False, callback=None, errback=None):
+        """
+        Search a zone for a given search query (e.g., for geological data, etc)
+
+        :param zone: NOT a string like loadZone - an already loaded nsone.zones.Zone, like one returned from loadZone
+        :return:
+        """
+        import nsone.zones
```

Comment:
Is this import necessary?

Reply:
it's too avoid a circular dependency, the other methods do similar